ice.c

```diff
@@ -3145,6 +3150,48 @@ void *janus_ice_send_thread(void *data) {
 			}
 			video_rtcp_last_sr = now;
 		}
+		/* Should we clean up old NACK buffers? */
+		if(now-last_nack_cleanup >= 500000) {
```

Comment:
I think you should base the cleanup frequency on `max_nack_queue`, I think a period of one quarter the `max_nack_queue_time` would be reasonable.